src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -21,11 +21,12 @@ import { BarChartResultsComponent } from "./BarChartResults.component";
 
 const dataStyle = css`
     margin: 0 auto;
-    display: grid;

```

Comment:
For info: The grid is not the real problem here. The problem is the `margin: auto` in centerChartStyle not working on grid-items. In general `margin: auto` is not the best way to center things with flex and grid, they have their own ways to handle centering. In the mibi-portal the only thing using margin: auto is the loading spinner, but it's because the animation needs it to work properly.